sdk/storage/azure-storage-file-share/azure/storage/fileshare/_file_client.py

```diff
@@ -283,12 +283,12 @@ def acquire_lease(self, lease_id=None, **kwargs):
 
         .. admonition:: Example:
 
-            .. literalinclude:: ../samples/blob_samples_common.py
-                :start-after: [START acquire_lease_on_blob]
-                :end-before: [END acquire_lease_on_blob]
+            .. literalinclude:: ../samples/file_samples_client.py
+                :start-after: [START acquire_lease_on_file]
+                :end-before: [END acquire_lease_on_file]
```

Comment:
Good catch! if this is changed the mark in file_samples_client.py should also be changed